### PR TITLE
[Draft] New Load Balancer object for HTTP Activity 4002 

### PR DESCRIPTION
**Pull request by nathanbvail**
#### Description of changes: 

- Created a new `Load Balancer` object that contains specific Load Balancer related information and added that object as an optional attribute in `HTTP Activity 4002`
- Created a new `Endpoint Responses` object that sits as an attribute inside of the `Load Balancer` object
- Created a new `endpoints` attribute that is an array of `Endpoint` objects
- Lastly, updated the dictionary with the necessary additions.

### CODE REVIEW

**Comment by nathanbvail** [comment]
Please provide any thoughts on the attribute names and the descriptions!

File: objects/endpoint_responses.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
+    "caption": "Endpoint Responses",
+    "name": "endpoint_responses",
```
Comment (Aniak5) [inline objects/endpoint_responses.json:3]: We should probably alter this to be a singular object with `Endpoint Response` / `endpoint_response`. With a singular `code` and a singular `endpoint`.  In the dictionary we will leave the `endpoint_responses` and set `"is_array": true`

File: dictionary.json [hunk 2]
```diff
@@ -1308,6 +1308,17 @@
       "description": "The end time of a time period. See specific usage.",
       "type": "timestamp_t"
     },
+    "endpoints": {
+      "caption": "Endpoints",
+      "description": "The Endpoint/s related to the entity.",
+      "is_array": true,
+      "type": "endpoint"
+    },
+    "endpoint_responses": {
```
Comment (Aniak5) [inline dictionary.json:1317]: Maybe we should consider renaming this to something that conveys that these are additional connection attempts.

File: dictionary.json [hunk 3]
```diff
@@ -1308,6 +1308,17 @@
...
+      "caption": "Endpoint Responses",
+      "description": "The Endpoint Responses attribute captures additional information about connection attempts to multiple endpoints",
```
Comment (Aniak5) [inline dictionary.json:1319]: We should add a `.` at the end of this description.

File: dictionary.json (hunk 2 above)
Comment (Aniak5) [reply dictionary.json:1311]: Per todays conversation `endpoint_connections`

File: dictionary.json [hunk 4]
```diff
@@ -1302,6 +1302,17 @@
       "description": "The end time of a time period. See specific usage.",
       "type": "timestamp_t"
     },
+    "endpoints": {
+      "caption": "Endpoints",
+      "description": "The Endpoint/s related to the entity.",
+      "is_array": true,
+      "type": "endpoint"
```
Comment (Aniak5) [inline dictionary.json:1309]: per our conversation today change this to be of type network_endpoint

**Pull request closed without merging**